mib.h

```diff
@@ -765,13 +765,13 @@ _icmpInParmProbs_obj = {
 	"icmpInParmProbs", 5, 0,
 	NULL, &_icmpInTimeExcds_obj
 },
-_icmpInSrcQuenchs_obj = {
-	"icmpInSrcQuenchs", 6, 0,
+_icmpInSrcQuenches_obj = {
+	"icmpInSrcQuenches", 6, 0,
```

Comment:
Happy to drop, I understand that individually each object is an `ICMP Source Quench`, but the standard English plural would be to use `-es` not `-s`.